### ISSUE TITLE
Fatal Error: Declaration of Illuminate\Http\Request::get must be compatible with Symfony\Component\HttpFoundation\Request::get

### ISSUE THREAD

**Issue by robrecord** [impress-org/givewp#6373]
## User Story

As a developer using Sage (a theme that uses laravel) and PHP 8, I want to install the plugin, but it gives a fatal error

## Details

```
Fatal error: Declaration of Illuminate\Http\Request::get(string $key, $default = null) must be compatible with Symfony\Component\HttpFoundation\Request::get($key, $default = null)
```

## Expected Behavior

Works.

## Steps to Reproduce

1. Install Sage theme https://roots.io/sage/
2. Install and activate GiveWP plugin
3. View error

## Visuals

<img width="1724" alt="Screenshot 2022-04-21 at 13 46 54" src="https://user-images.githubusercontent.com/38916/164461667-8ed37017-f5bf-4f1f-be03-826bb82a1807.png">

## Additional Context

This looks like a mismatch of Laravel/Symphony versions

## System Information

<details><summary>Details</summary>

PHP 8
Laravel 9
Sage v10.1.6
Bedrock
Trellis

</details>

## Acceptance Criteria

-   [x] Something happens when an action is taken.
-   [x] Something does not happen when an action is taken.
-   [x] Fixing behavior in Component A does not affect existing behavior in Component B.

**Comment by robrecord** [comment]
I think to fix this it would require the dependency symfony/http-foundation to be updated to ^6.0

**Comment by robrecord** [comment]
Confirmed that requiring `"symfony/http-foundation:^5.4"` fixes the issue.

Shall I submit a pull request?

**Comment by jonwaldstein** [comment]
@robrecord unfortunately we can't upgrade that package because we have to support php 5.6 at the moment.  I'll discuss with the team to see if there's anything we can do to prevent these types of conflicts.

**Comment by JasonTheAdams** [comment]
We've been looking at using [Strauss](https://github.com/BrianHenryIE/strauss) within GiveWP to prefix certain vendor namespaces and avoid such conflicts. This is most likely what we'll do.

**Comment by robrecord** [comment]
Thank you. 

I looked at namespacing with composer; for my use case for pretty much need to sort this out so I’ll make a fork with the affected packages namespaced using Strauss or php-scoper. If you like it maybe you could take a pull-request.

**Comment by robrecord** [comment]
I managed to get it mostly working by upgrading the symphony package and removing faker (required for PHP 8), although the result of that is I get fatal errors when using wp-cli so I need to go down the scoping route. Will let you know how it goes.

**Comment by robrecord** [comment]
I attempted and failed to get Strauss to work; its latest version v0.11 requires PHP 7, and the version compatible with PHP 5.6 (v0.3) uses a completely different mechanism which I believe would require rewriting a lot of `namespace` references, which I decided not to do.

Instead I present my solution to the issue in the hope that it may help someone in future:

In `composer.json`, edit scripts.bundle to `rsync -rc --exclude-from=.distignore ./ release/ --delete --delete-excluded` - note the added forward slash after the period.

Then:
```
composer config platform.php "7.2.5"
composer require "symfony/http-foundation:^5.4"
composer update --lock --no-dev
composer bundle
rsync -rc --exclude-from=.distignore release/ ../your-site/wp-content/plugins/give/ --delete --delete-excluded
```

Hopefully I haven't missed any steps; this allowed the plugin to work for me.
